sql/option.sh

```diff
@@ -165,32 +165,54 @@ echo ''
 echo '###### COMMAND OPTION TEST-0017 ######'
 echo '###### invalid value in pg_rman.ini ######'
 init_catalog
-echo "SMOOTH_CHECKPOINT=FOO" >> ${BACKUP_PATH}/pg_rman.ini
```

Comment:
I think it's better not to change this line and change line 190 because the number of diff became minimum.